playbooks/acs.yml

```diff
@@ -318,6 +326,16 @@
   tags:
     - search_enterprise
 
+- name: Populate adf app facts
+  hosts: acc:adw
+  gather_facts: false
+  tasks:
+    - name: Populate adf app config when identity available
+      when: identity_enabled
+      ansible.builtin.set_fact:
+        adf_app_oauth_configuration:
+          APP_CONFIG_AUTH_TYPE: oauth
+
```

Comment:
So we rely on the fact the default config match the initial use case?
I think adding some more params to later on suport external identity is relatively cheap, isn't it?

Reply:
yes, and I remember we need those defaults present because the packaged configuration contains $placeholders which must be replaced to have a working install (even when sso is disabled).

I think I could repeat here the properties which probably needs to be changed when using an external identity:
```
APP_CONFIG_IDENTITY_HOST: "{protocol}//{hostname}{:port}/auth/realms/alfresco"
APP_CONFIG_OAUTH2_HOST: "{protocol}//{hostname}{:port}/auth/realms/alfresco"
APP_CONFIG_OAUTH2_CLIENTID: alfresco
```